corehq/apps/reports/v2/models.py

```diff
@@ -59,6 +74,15 @@ def context(self):
             'slug': self.slug,
             'endpoints': [e._asdict() for e in endpoints],
             'columns': [c._asdict() for c in self.columns],
+            'column_filters': [
+                dict(
+                    filterType=f.filter_type,
```

Comment:
nit: I think we try to convert to `camelCase` in javascript when loading the initial page data, especially since there are mixed cases here (`column_filters` and `filterType`) and you use `filterType` to fetch the value in python below. 

I'm also not a huge fan of this `dict` syntax, but I can't quite articulate why.